crypto/txscript/Cargo.toml

```diff
@@ -12,14 +12,19 @@ repository.workspace = true
 [[example]]
 name = "kip-10"
 
+[[example]]
+name = "script-viewer"
+
 [features]
 wasm32-core = []
 wasm32-sdk = []
 
 [dependencies]
 blake2b_simd.workspace = true
 borsh.workspace = true
+bs58.workspace = true
 cfg-if.workspace = true
+hex.workspace = true
```

Comment:
use faster hex. i think we should use the library everywhere instead of mixing them

Reply:
addressed in [review: replace hex with faster hex for fresh code](https://github.com/kaspanet/rusty-kaspa/pull/743/commits/4a0a6c26faf7be6bd9a5162b5e7fcc4af0cb52f0)